### PR TITLE
Issue 62

### PR DESCRIPTION
**Pull request by remingtontr**
New article created.

### CODE REVIEW

File: docs/hardware-interfaces/ethernet.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  </TabItem>
+</Tabs>
+
+See the [Omega2S hardware design guide](https://github.com/OnionIoT/Omega2/blob/master/Documents/Omega2S%20Hardware%20Design%20Guide.pdf) for more information and recommendations on designing boards with Ethernet.
+
+## Software
+The Ethernet port can be configured to be a client or a Dynamic Host Configuration Protocol (DHCP) host. It is set to client mode by default. 
+
+For more information on Ethernet networking, see the [Ethernet networking article](https://documentation.onioniot.com/networking/ethernet).
```
Comment (greenbreakfast) [inline docs/hardware-interfaces/ethernet.md:36]: Please make this a relative link, that way the compilation takes care of the base URL
Example relative link: https://github.com/OnionIoT/documentation/blob/main/docs/hardware-interfaces/gpio.md?plain=1#L45

Please also add this to the style guide

**Review by greenbreakfast — COMMENTED** [review]
@remingtontr, I have one suggestion regarding the link to the ethernet networking article, see below

**Comment by remingtontr** [comment]
Both these fixes have been done and pushed.

Tom Remington
________________________________

**Comment by greenbreakfast** [comment]
merging, cc #62

**Pull request merged**